src/include/common/file_system/local_file_system.h

```diff
@@ -27,6 +27,9 @@ struct LocalFileInfo : public FileInfo {
 
 class KUZU_API LocalFileSystem final : public FileSystem {
 public:
+    LocalFileSystem() = default;
+    explicit LocalFileSystem(std::string homeDir) : FileSystem(std::move(homeDir)) {}
```

Comment:
I think it's because we are using creating dummy local file system just to use its member function in many place? We can change to pass in empty string as homeDir in such usages.

Reply:
removed this default constructor